ui/src/containers/CreateVolume.js

```diff
@@ -376,8 +376,7 @@ const CreateVolume = (props) => {
       yup.object().shape({
         name: yup
           .string()
-          .matches(volumeNameRegex, intl.translate('name_error'))
-          .required(''),
+          .matches(volumeNameRegex, intl.translate('name_error')),
```

Comment:
Why is this required in your PR?

Reply:
The cypress tests were failing after moving to webpack 5. Likely a side effect of the dependencies bump I ran. The consequence was that the button remained disabled after having correctly filled the form because the volumes.name was not filled when creating a single volume.